simplivity/resources/resource.py

```diff
@@ -223,18 +223,22 @@ def do_get(self, uri):
         """
         return self._connection.get(uri)
 
-    def do_post(self, uri, data, timeout, custom_headers):
+    def do_post(self, uri, data, timeout, custom_headers, filters=None):
```

Comment:
Can we call it as flags instead of filters?

Reply:
Hi Sijeesh,
We were planning to name it as queryParams, but just to be consistent with do_get method we named this as filters, as this will work on specific object.

Please let me know if you still want me to change the parameter name?

Reply:
For do_post will be either specifying parameters such as cluster_group_id, or flags such as overwrite = true.

For do+get they are used as a way to filter down the data being retrieved.

filters being renamed to flags makes sense here

Reply:
Renamed the filters to flags as per the above comments.